qjsc.c

```diff
@@ -390,39 +424,83 @@ int main(int argc, char **argv)
     namelist_add(&cmodule_list, "os", "os", 0);
     namelist_add(&cmodule_list, "bjson", "bjson", 0);
 
-    for(;;) {
-        c = getopt(argc, argv, "ho:N:Cmn:bxesvM:p:S:D:");
-        if (c == -1)
-            break;
-        switch(c) {
-        case 'h':
-            help();
-        case 'b':
-            output_type = OUTPUT_RAW;
-            break;
-        case 'o':
-            out_filename = optarg;
-            break;
-        case 'e':
-            output_type = OUTPUT_C_MAIN;
-            break;
-        case 'n':
-            script_name = optarg;
+    while (optind < argc && *argv[optind] == '-') {
```

Comment:
Stopping on `-` is maybe not useful for qjsc now, but I decided to keep it for consistency with qjs.

Potentially useful in the future to bake in arguments to the compiled script somehow?